packages/@aws-cdk-testing/framework-integ/test/aws-synthetics/test/integ.canary.ts

```diff
@@ -100,10 +100,12 @@ const puppeteer52 = createCanaryByRuntimes(Runtime.SYNTHETICS_NODEJS_PUPPETEER_5
 const puppeteer62 = createCanaryByRuntimes(Runtime.SYNTHETICS_NODEJS_PUPPETEER_6_2);
 const puppeteer70 = createCanaryByRuntimes(Runtime.SYNTHETICS_NODEJS_PUPPETEER_7_0);
 const puppeteer80 = createCanaryByRuntimes(Runtime.SYNTHETICS_NODEJS_PUPPETEER_8_0);
+const playwright10 = createCanaryByRuntimes(Runtime.SYNTHETICS_NODEJS_PLAYWRIGHT_1_0);
```

Comment:
Could you add tests for `SYNTHETICS_NODEJS_PUPPETEER_9_0` and `SYNTHETICS_NODEJS_PUPPETEER_9_1`? They weren't covered when they were added

Reply:
I've added!